### PR TITLE
added tadf to assetlist.json  

### PR DESCRIPTION
**Pull request by Fidel2001**

### CODE REVIEW

File: terra/assetlist.json [hunk 1]
```diff
@@ -5732,6 +5732,32 @@
         }
       ]
     },
+    {
```
Comment (JeremyParish69) [inline terra/assetlist.json:5735]: Please move this new asset to the bottom of the list

File: terra/assetlist.json [hunk 2]
```diff
@@ -5732,6 +5732,32 @@
...
+      "description": "Tadpole Frogztrik Token meme",
+      "type_asset": "cw20",
+      "address": "terra19gzvkw5hft3ulp8n6ws3p9yradhc802axqal59lzxau6rqxktj9q0dwf9c",
+      "denom_units": [
+        {
+          "denom": "cw20:terra19gzvkw5hft3ulp8n6ws3p9yradhc802axqal59lzxau6rqxktj9q0dwf9c",
+          "exponent": 0
+        },
+        {
+          "denom": "tadf",
```
Comment (JeremyParish69) [inline terra/assetlist.json:5745]: please define a display denom. I assume it should just be "tadf"

File: terra/assetlist.json [hunk 3]
```diff
@@ -5758,6 +5758,32 @@
...
+      "symbol": "TADF",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/tadf.png"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/tadf.png"
+      }
+    },
```
Comment (JeremyParish69) [inline terra/assetlist.json:5787]: this comma breaks the schema, please remove.

**Comment by Fidel2001** [comment]
Fixed

File: terra/assetlist.json (hunk 2 above)
Comment (JeremyParish69) [reply terra/assetlist.json]: Reminding you that we are still missing "display"

File: terra/assetlist.json [hunk 4]
```diff
@@ -5758,6 +5758,60 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/rakoff.png"
         }
       ]
+    },
+    {
+      "description": "Tadpole Frogztrik Token meme",
+      "type_asset": "cw20",
+      "address": "terra19gzvkw5hft3ulp8n6ws3p9yradhc802axqal59lzxau6rqxktj9q0dwf9c",
+      "display": "tadf"
```
Comment (JeremyParish69) [inline terra/assetlist.json:5766]: this needs a comma after it

File: terra/assetlist.json [hunk 5]
```diff
@@ -5758,6 +5758,60 @@
...
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/tadf.png"
+      }
+    },
+    {
+      "description": "Frogztrik NFT Token",
+      "type_asset": "cw20",
+      "address": "terra1wez9puj43v4s25vrex7cv3ut3w75w4h6j5e537sujyuxj0r5ne2qp9uwl9",
+      "display": "frog"
```
Comment (JeremyParish69) [inline terra/assetlist.json:5793]: need comma

**Pull request merged**